stripe/build.gradle

```diff
@@ -1,9 +1,17 @@
 apply plugin: 'com.android.library'
 
+configurations {
```

Comment:
Actual changes begin again down here.
Adding this configuration is to make travis happy with our javadoc builds so we can ship the code.